include/unifex/any_sender_of.hpp

```diff
@@ -70,6 +71,11 @@ struct _rec_ref<CPOs, Values...>::type
     : _rec_ref_base<CPOs>::template type<Values...>(*op)
     , stoken_(st) {}
 
+  template<typename Error>
+  void set_error(Error &&error) const noexcept {
```

Comment:
Having to add this sort of thing here makes me wonder whether it should be built in to the `set_error()` CPO itself...

e.g. try to call `set_error(r, e)` and if that fails then if you can call `set_error(r, exception_ptr)` and you can call `as_exception_ptr(e)` then do that, otherwise fail to find an overload.

Reply:
This is what I did in the last update.
Now, `set_error()` handles this directly.
If the receiver accepts `std::exception_ptr` as error type and the given `Error` is convertible using `as_exception_ptr`, then it'll use it.
It also results in rollback of all `xxx::set_error<Error>(...)` previous modifications.